src/Propel/Common/Config/ConfigurationManager.php

```diff
@@ -8,9 +8,11 @@
 
 namespace Propel\Common\Config;
 
+use Exception;
```

Comment:
Where does this one come from?

Reply:
Oh, initially, the catch was for all exceptions, and a new PropelException was thrown. But the tests relied on InvalidConfigurationException, so I changed it back. Forgot to remove the import - thank you for finding!